src/main/java/org/opensrp/queue/QueueHelper.java

```diff
@@ -0,0 +1,24 @@
+package org.opensrp.queue;
+
+import org.opensrp.service.PlanService;
+import org.smartregister.domain.Jurisdiction;
+import org.smartregister.domain.PlanDefinition;
+import org.smartregister.pathevaluator.TriggerType;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class QueueHelper implements org.smartregister.pathevaluator.dao.QueuingHelper {
+
+	@Autowired
+	private PlanService planService;
```

Comment:
remove private here as well to help unit tests

Reply:
Done